Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -206,12 +206,22 @@ void Settings::SetStateSlot(int slot)
 
 bool Settings::GetRenderToMain() const
 {
-  return value(QStringLiteral("Graphics/RenderToMain"), false).toBool();
+  return SConfig::GetInstance().bRenderToMain;
+}
+
+void Settings::SetRenderToMain(bool rendertomain)
```

Comment:
To have a wrapper around any ``SConfig`` code, so when the system is updated / changed only ``Settings.cpp`` is affected